features/ALS010_Alignment-segment-shape-representation-has-exactly-one-item.feature

```diff
@@ -0,0 +1,23 @@
+@ALS
+@version1
+Feature: ALS010 - Alignment segment shape representation must have exactly one item
+The rule verifies that IfcAlignmentSegment is represented correctly with representation type Segment and a single item.
+
+Background:
+    Given A model with Schema "IFC4.3"
+    Given An IfcAlignmentSegment
+    Given Its attribute Representation
+    Given Its attribute Representations
+
+  @E00020
+  Scenario: Agreement on each IfcAlignmentSegment using correct representation - Type
+
+      Given Its attribute RepresentationType
+      Then The value must be "placeholder_invalid"
+
+
+  @E00040
+  Scenario: Agreement on each IfcAlignmentSegment having correct number of representation items
+
+      Given Its attribute Items
+      Then There must be one representation item
```

Comment:
I think we should remove the attribute name from the statement in order to make it more general.

```suggestion
      Then There the number of values should be 1
```

It's a bit of similar predicate to https://github.com/buildingSMART/ifc-gherkin-rules/blob/development/features/steps/thens/values.py#L31-L47 except there we count the occurrences of a certain value in the list. Here we only count the items in the list.

Reply:
This has been revised based on today's discussion.  As for making this step more general, I would prefer to leave it as-is, in the spirit of YAGNI.